models/daily_spend.sql

```diff
@@ -177,13 +145,15 @@ automatic_clustering_spend_daily as (
         null as storage_type,
         null as warehouse_name,
         null as database_name,
-        coalesce(sum(stg_metering_history.credits_used * cloud_services_rates.effective_rate), 0) as spend
+        coalesce(sum(stg_metering_history.credits_used * daily_rates.effective_rate), 0) as spend
     from dates
     left join {{ ref('stg_metering_history') }} on
         dates.date = convert_timezone('UTC', stg_metering_history.start_time)::date
         and stg_metering_history.service_type = 'AUTO_CLUSTERING'
-    left join cloud_services_rates on
-        dates.date = cloud_services_rates.date
+    left join {{ ref('daily_rates') }}
+        on dates.date = daily_rates.date
+            and daily_rates.service_type = 'COMPUTE'
+            and daily_rates.usage_type = 'automatic clustering'
```

Comment:
@NiallRees FYI, you were using the `cloud services` rate for `automatic clustering`, `snowpipe` and `query acceleration`, but each of them get their own rate in `rate_sheet_daily`.